pages/app/index.html

```diff
@@ -4,6 +4,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- This favicon gets overritten via webpack-->
```

Comment:
Ran locally, there are two icons now
<img width="556" alt="image" src="https://github.com/user-attachments/assets/76e7b5f9-e7ae-4fad-ba32-7659dcec102e">

So, the icon does not get overwritten and also the new icon is after the CSP generator, so there are CSP errors in your output too

Reply:
When there is 2, this gives the browsers the ability to pick a winner.  So one webpack and one base64 as backup? I will remove the non webpack. I am not able to reproduce the CSP error locally
<img width="527" alt="Screenshot 2024-08-28 at 15 08 00" src="https://github.com/user-attachments/assets/dabca247-ddf7-40a1-8445-d976d8396ffa">

Reply:
I do not think you can control the injection order in the plugin, it always appends stuff to the end of the `<head>`.

So, the `favicon` prop won't work, it needs manual template:

```
<link rel="icon" type="image/png" sizes="16x16" href="data:;<%= htmlWebpackPlugin.options.faviconBase64 %>" />
```

and then pass it via

```
new HtmlWebpackPlugin({
   ...
  faviconBase64: fs.readFileSync('favion.ico', 'base64')
}),
```